### PR TITLE
Skip report vv warn if no sybase passwd file for db

### PR DESCRIPTION
**Pull request by jeanconn**
## Description

Skip report vv warn if no sybase passwd file for db

## Testing

- [x] Passes unit tests on MacOS, linux, Windows (at least one required)
- [x] Functional testing
   - ran as jeanconn and kadi user on Linux from production ska3 and see test pass and test skip as appropriate

Fixes #

### CODE REVIEW

**Comment by jeanconn** [comment]
More klunky compromise here, but I think this keeps me happy about test coverage and could stop the warns (though now @javierggt would need to *delete* his passwd file if it isn't actually working for test to pass... and put it back when it works).

File: mica/report/tests/test_write_report.py [hunk 1]
```diff
@@ -9,16 +9,21 @@
 
 from .. import report
 
+user = os.environ.get('USER') or os.environ.get('LOGNAME')
```
Comment (taldcroft) [inline mica/report/tests/test_write_report.py:12]: I think the best way to get user information is this, which you can reliably use on any system:
```
import getpass
getpass.getuser()
```

File: mica/report/tests/test_write_report.py [hunk 2]
```diff
@@ -9,16 +9,21 @@
...
     with Ska.DBI.DBI(server='sqlsao', dbi='sybase', user=user, database='axafvv') as db:
         HAS_SYBASE_ACCESS = True
 except Exception:
-    if on_head_network() and not has_sybase():
-        warn("On HEAD but no sybase access. Run ska_envs or define SYBASE/SYBASE_OCS")
     HAS_SYBASE_ACCESS = False
 
+    # If the user should have access, warn about the issue.
+    if (on_head_network() and not has_sybase() and
+            os.path.exists(os.path.join(os.environ['SKA'], 'data', 'aspect_authorization',
```
Comment (taldcroft) [inline mica/report/tests/test_write_report.py:23]: You should just start using `Path` at every opportunity. It is very much nicer in almost every case:
`Path(os.environ['SKA'], 'data', 'aspect_authorization', f'sqlsao-axafvv-{user}').exists()`.

**Review by taldcroft — CHANGES_REQUESTED** [review]
Both of the changes are not required, but are about learning (through usage) new and better patterns for doing common things.

**Comment by jeanconn** [comment]
Sure.  Yes, I was just extending what was there (at least in my mind) instead of doing it a little better.  I can do it a little better.

**Comment by jeanconn** [comment]
@javierggt up to you if you want to leave your sqlsao-axafvv-* file around or if you want to delete so the tests will pass against master until vv access actually works.

**Pull request merged**